service-base/src/main/java/org/eclipse/hono/service/tracing/SamplerProducer.java

```diff
@@ -195,18 +195,12 @@ Resource resource() {
      */
     @Singleton
     @Produces
-    @IfBuildProperty(name = "quarkus.opentelemetry.enabled", stringValue = "true")
-    Sampler sampler(final TracerRuntimeConfig tracerRuntimeConfig) {
-        final var samplerConfig = tracerRuntimeConfig.sampler;
-        if (samplerConfig.parentBased) {
-            LOG.warn("""
-                    'quarkus.opentelemetry.tracer.sampler.parent-based' set to 'true' - \
-                    custom Hono Sampler will not be applied to child spans""");
-        }
-        final String samplerName = Optional.ofNullable(otelConfig.getString(PROPERTY_OTEL_TRACES_SAMPLER))
-                .orElse(samplerConfig.samplerName);
+    @IfBuildProperty(name = "quarkus.otel.enabled", stringValue = "true")
```

Comment:
There is no `quarkus.otel.enabled` property defined (see the [config reference](https://quarkus.io/guides/opentelemetry#configuration-reference)). There is now a `quarkus.otel.sdk.disabled` property, but that is a runtime property.
Therefore this line should be removed.

Reply:
Done.